github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,32 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
+    "Entry point for the workflow"
```

Comment:
i think it's customary to have `"""` for docstrings. https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
i would also either remove the docstring or put something more descriptive.